newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -145,6 +157,20 @@ export default class ExtensionInstallDialog extends Component<Props, State> {
               <Text noMargin size="body2">
                 <Trans>Version {' ' + extensionShortHeader.version}</Trans>
               </Text>
+              <div style={styles.authors}>
+                {extensionShortHeader.authors && (
+                  <>
+                    {extensionShortHeader.authors.map(author => (
+                      <Chip
+                        size="small"
+                        style={styles.chip}
+                        label={author.username}
+                        key={author.username}
+                      />
```

Comment:
I know it's early, but I would move this already into a `<UserPublicProfileChip user={author} />`

This will give the opportunity to later add the avatar everywhere at once, customize the color, etc.. according to the UserPublicProfile